server/user_cloud.go

```diff
@@ -32,10 +30,7 @@ func httpACUserRedirect(jci *jiraCloudInstance, w http.ResponseWriter, r *http.R
 		return http.StatusBadRequest, err
 	}
 
-	submitURL := path.Join(jci.Plugin.GetPluginURLPath(), routeACUserConnected)
-	if requireUserApproval {
-		submitURL = path.Join(jci.Plugin.GetPluginURLPath(), routeACUserConfirm)
-	}
+	submitURL := path.Join(jci.Plugin.GetPluginURLPath(), routeACUserConfirm)
```

Comment:
The confirmation is required to be secure. Otherwise you can trick people into clicking the link.

Reply:
@jasonblais is requesting the opposite :) Can you two please resolve it? (I'm 3/5 that security should win, and we should have the explicit warning/confirmation)

Reply:
Yup. Will chat with @jasonblais

Reply:
Chatted with @crspeller, given the level of a security vulnerability if this button wasn't added, I'm good with this change. 

Approving PR.